project/liamlin5566/README.md

```diff
@@ -0,0 +1,72 @@
+# NSD Project: Kernel K-means on Image
+
+### Basic Information
+This project is the implementation of Kernel K-means on image with c++ and python. I will also try to accelerate the computation on GPU by using CUDA if time is enough.
+
+### Problem to Solve
+I will do clustering on image data by using kernel kmeans, which "similar" pixels would be grouped together. 
+
+Kernel k-means is well-known unsupervised clustering algorithm. It will find k cluster centers to represent each group, and assign each data point to its nearest center like traditional k-means. Different from traditional k-means using euclidean distance, kernel kmeans utilize kernel function to calculate distance:

```

Comment:
Please use a reasonable line width so that I can add comment at proper location.  See #91 for an example.